tests/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
```

Comment:
could we just use the docker-compose.yaml from the docker folder and add anvil there? Would also be happy with replacing the other one with this.

Reply:
The `docker-compose.yml` from uses very specific and somewhat idiosyncratic port numbers that I chose to that they would not conflict with other things a dev might have running. I think it's easier to keep them as separate files.